draft-irtf-qirg-principles-03.xml

```diff
@@ -795,6 +819,38 @@
            information via separate classical channels which the repeaters will
            have to correlate with the qubits stored in their memory.</t>
 
+           <t> First generations of quantum networks will not be store-and-forward
+           networks.
+           <vspace blankLines="1" />
+
+           As described in <xref target="elg"></xref>,
+           Quantum links provide Bell pairs that are undirected network
+           resources, in contrast to directed frames of classical networks.
+           This phenomenological distinction should result in architectural
+           differences between quantum networks and classical networks;
+           quantum networks put link Bell pairs together to an end-to-end
+           Bell pair where classical networks deliver messages from an end
+           to the other end hop by hop.
+           <vspace blankLines="1" />
+
+           Classical networks receive data on one interface, store it on local buffers, 
+           then forward the data to another appropriate interface.
+           Quantum networks store Bell pairs and then
+           execute entanglement swapping instead of forwarding in the data plane.
+           Such quantum networks are "store and swap" networks.
+           In "store and swap" networks, we do not need to care about
+           the order in which the Bell pairs were generated since they are
+           undirected. This distinction makes control algorithms and
+           optimization of quantum networks different from classical ones.
+           <vspace blankLines="1" />
+
+           It is worth noting, that third generation <refer to generations section>
+           quantum repeaters will be able to transmit qubits directly
+           in a "store and forward" manner since they will be able to carry
+           arbitrary states hop by hop. Thus, the networking algorithms of the
+           later generations of quantum networks may end up having more in 
+           common with classical networks.</t>
+           
```

Comment:
So I've been thinking about this section. Currently it suggests that 3G will likely do store and forward. I think this needs to be relaxed a bit. Or at least made consistent with what I will soon propose with regards to the three generations for the draft. Can we move this to the other pull request too? Rather than merging it now and then changing it. 

Let me know, if you're okay with this and I'll just remove this myself. You'll need to wait for me to merge this request first I guess before you update the other PR.

Reply:
No problem. You can remove those sentences by yourself. Please go ahead :)
I'll update the other PR after you merge this request.

Reply:
```suggestion
```